autospec/specfiles.py

```diff
@@ -239,12 +239,13 @@ def write_files_header(self):
         groups["lib"] = "Libraries"
         groups["doc"] = "Documentation"
         groups["data"] = "Data"
+        groups["services"] = "Systemd services"
 
         deps = {}
         deps["dev"] = ["lib", "bin", "data"]
         deps["doc"] = ["man"]
         deps["dev32"] = ["lib32", "bin", "data", "dev"]
-        deps["bin"] = ["data", "libexec", "config", "setuid", "attr", "license", "man"]
+        deps["bin"] = ["data", "libexec", "config", "setuid", "attr", "license", "man", "services"]
```

Comment:
I'm actually not sure about having bin pull in the services but this probably is the right thing to try first.